tests/conditional_compilation/test_collect.py

```diff
@@ -9,11 +9,34 @@
 import glob
 import os
 import sys
+import pytest
+from inspect import getsourcefile
+from pathlib import Path
 
 from proc_utils import cmd_exec  # pylint: disable=import-error
+from tests_utils import write_session_info, SESSION_INFO_FILE
 
 
-def test_cc_collect(test_id, model, sea_runtool, benchmark_app, collector_dir, artifacts, test_info):
+@pytest.fixture(scope="function")
+def test_info(request, pytestconfig):
+    """Fixture function for getting the additional attributes of the current test."""
+    setattr(request.node._request, "test_info", {})
+    if not hasattr(pytestconfig, "session_info"):
+        setattr(pytestconfig, "session_info", [])
+
+    yield request.node._request.test_info
+
+    pytestconfig.session_info.append(request.node._request.test_info)
+
+
+@pytest.fixture(scope="session")
+def save_session_info(pytestconfig, artifacts):
```

Comment:
`save_session_info` isn't used now because `autouse=True` was removed. Please, add `save_session_info` to `test_cc_collect` def as function argument to enable fixture